relay-sampling/src/lib.rs

```diff
@@ -357,6 +358,25 @@ impl TimeRange {
     }
 }
 
+/// A sampling strategy definition.
+///
+/// A sampling strategy refers to the strategy that we want to use for sampling a specific rule.
+#[derive(Debug, Clone, Copy, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+#[serde(tag = "type")]
+pub enum SamplingStrategy {
```

Comment:
This might be subjective but I would rather call this `SamplingValue` or even `SampleRate`, rather than `Strategy`.

Reply:
I would prefer `SamplingValue` thanks for the suggestion!